docs/source/user-guide/cli.md

```diff
@@ -194,6 +194,19 @@ DataFusion CLI v16.0.0
 2 rows in set. Query took 0.007 seconds.
 ```
 
+You can also query directly from the remote location via HTTP(S) without
```

Comment:
🎉 

I left a suggestion where hopefully this could be generalized to "you can also query directly from any remote location ..."

Reply:
I'm glad I kinda followed a right path here. As you saw @alamb, I left a `TODO` comment there to implement this for other remote resources (e.g. `s3:/`) for couple of reasons:
1. the issue talks about http(s) specifically so I didn't want to expand the scope of the RP, (more chances for it to get merged too 😁 ) and cover the other protocols in a follow up PR
2. I didn't want to modify https://github.com/apache/arrow-datafusion/blob/10ae9343368a893012aa80b66c02d45b4f461f9f/datafusion-cli/src/exec.rs#L305-L339 as I'd have to lift it to pub(crate) level, and more importantly, I didn't have access to command `options` (those are needed for S3 etc auth) in the place I modified the code, so I need to figure out how to get those if they are provided by the executions context (I could just use an empty HashMap so the config values are picked from the environment but that would be limilted)

If it's OK I suggest I (or someone else) addresses additional remote protocols in a follow up PR, I an create a relevant issue.